Source/Fuse.Maps/iOS/MapViewDelegate.m

```diff
@@ -214,6 +224,48 @@ -(void) clearMarkers
 			[_mapView removeAnnotation:a];
 		}
 		[_annotations removeAllObjects];
+
+		for(id key in _overlays)
+		{
```

Comment:
Yes, and a space between "for" and the left paren. That would make this code consistent with the surrounding code.

Our [rule](https://github.com/fusetools/fuselibs-public/blob/master/Documentation/CodingStyle.md#uno--c) against egyptian braces only applies to Uno-code.